### ISSUE TITLE
Sharing common state between filters

### ISSUE THREAD

**Issue by conqerAtapple** [facebook/wangle#149]
Hello Wanglers
 If I had to share some common state between filters in the Pipeline what is the recommended way to do this? Say, the first filter unwraps a network packet which contains header with information about the entire connection duration that can be shared between different filters, how do I extract this information/header in one filter and populate a map or something that can be used by other filters.

thanks

**Comment by graphcareful** [comment]
Yes I would like to know the same thing. In more then one place in the documentation the following is stated: 

```When events are called, a Context* object is passed to the Handler - this means state can be stored in the context object, and a single instantiation of any individual Handler can be used for the entire program.```

, however a concrete example is never shown.
